concepts/05 UI Components/Form/01 Getting Started with Form/80 Submit the Form.md

```diff
@@ -0,0 +1,268 @@
+To submit a form, add the [Button Item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/ButtonItem/) and set the [useSubmitBehavior](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#useSubmitBehavior) property to **true**. The Form can be submitted to a server only when all the validation checks pass successfully. 
+
+In this tutorial, we use the <a href="https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout" target="_blank">setTimeout</a> function to emulate the form submission. The Form is wrapped in the HTML <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form" target="_blank">form</a> element. We set the <a href="https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault" target="_blank">preventDefault</a> property to **true** to override the HTML form submit event:
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#form").dxForm({
+            formData: {
+                // ...
+            },
+            colCount: 2,
+            items: [{
+                itemType: "group",
+                caption: "Personal Information",
+                colCount: 2,
+                items: [{
+                    dataField: "name",
+                    isRequired: "true"
+                }, {
+                    dataField: "officeNumber",
+                    validationRules: [{
+                        type: "numeric"
+                    }] 
+                }, {
+                    dataField: "email",
+                    validationRules: [{
+                        type: "email"
+                    }] 
+                }]
+            }, {
+                itemType: "button",
+                buttonOptions: {
+                    text: "Submit the Form",
+                    useSubmitBehavior: true
+                }
+            }]
+        });
+
+        $("#form-container").on("submit", function(e) {
+            setTimeout(function () { 
```

Comment:
Let's validate the form here.

Reply:
Did I get it right and you suggest to replace the validation via the button's useSubmitBehavior with the validate() method?